src/e2e/step-definitions/common.ts

```diff
@@ -2,11 +2,13 @@
  * SPDX-FileCopyrightText: 2023 Lifely
  * SPDX-License-Identifier: EUPL-1.2+
  */
-import { Given, When, Then } from "@cucumber/cucumber";
+import { Given, Then } from "@cucumber/cucumber";
 import { CustomWorld } from "../support/worlds/world";
+import { worldPossibleZacUrls } from "../utils/schemes";
 
-Given("{string} navigates to {string}", { timeout: 60 * 1000 }, async function (this: CustomWorld, user, url) {
-    await this.openUrl(url);
+Given("{string} navigates to {string} with path {string}", { timeout: 60 * 1000 }, async function (this: CustomWorld, user, urlKey, path) {
```

Comment:
Can we add types to the function parameters. We want the scenarios to fail if they are setup incorrectly, no?

Reply:
the attributes in the function are automatically typed to a string